test/Dockerfile.bionic

```diff
@@ -94,3 +94,6 @@ RUN install-tool terraform 1.0.11
 
 # renovate: datasource=github-releases lookupName=git-lfs/git-lfs
 RUN install-tool git-lfs v3.0.2
+
+# renovate: datasource=github-releases lookupName=jsonnet-bundler/jsonnet-bundler
+RUN install-tool jb v0.4.0
```

Comment:
Please also add a new test to `test/jb/Dockerfile` (needs to be created) where you call jb on a small sample file.

So we can verify it works as expected